### PR TITLE
fix: sonar bug: 105,106,107,153,154,155,156,157,158,159,160,161,162,1…

### PR DESCRIPTION
**Pull request by xiaohei417**
…63,164,175,176,177,178,179,180,181,182,183,184,209,210,212,213

### CODE REVIEW

**Review by jxinging — DISMISSED** [review]

File: portal/apps/task.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ func TaskOutput(c *ctx.ServiceContext, form forms.DetailTaskForm) (interface{},
 	if err != nil && err.Code() == e.TaskNotExists {
 		return nil, e.New(e.TaskNotExists, err, http.StatusNotFound)
 	} else if err != nil {
-		c.Logger().Errorf("error get task by id, err %s", err)
+		c.Logger().WithField("task", "output").Errorf("%s", err)
```
[the remark this answers is not in the document]
Comment (xiaohei417) [reply portal/apps/task.go]: done

File: portal/libs/db/mysql_gorm.go [hunk 2]
```diff
@@ -272,7 +272,7 @@ func (s *Session) Exists() (bool, error) {
 }
 
 func (s *Session) autoLazySelect() *Session {
-	selects, ok := s.db.Get("app:lazySelects")
+	selects, ok := s.db.Get("app:lazySelects") //nolint
```
[the remark this answers is not in the document]
Comment (xiaohei417) [reply portal/libs/db/mysql_gorm.go]: done

File: portal/apps/template.go [hunk 3]
```diff
@@ -373,7 +373,7 @@ func DeleteTemplate(c *ctx.ServiceContext, form *forms.DeleteTemplateForm) (inte
 	}
 
 	if err := vcsrv.SetWebhook(vcs, tpl.RepoId, token.Key, []string{}); err != nil {
-		c.Logger().Errorf("set webhook err :%v", err)
+		c.Logger().WithField("action", "set webhook").Errorf("%v", err)
```
[the remark this answers is not in the document]
Comment (xiaohei417) [reply portal/apps/template.go:376]: done

**Review by jxinging — CHANGES_REQUESTED** [review]
又有冲突了 😭

**Pull request merged**